quickstarts/maven/docker-file-simple/README.md

```diff
@@ -0,0 +1,99 @@
+[INFO] ------------------------------------------------------------------------
+$ docker images | grep docker-file-simple
+jkube/docker-file-simple                  1.0                         c8df6da232b3        8 seconds ago       516MB
+```
+
+# Generating Kubernetes Manifests and Deploying to Kubernetes
+```
+$ mvn k8s:resource k8s:apply
+[INFO] Scanning for projects...
+[INFO] 
+[INFO] ----------------< org.eclipse.jkube:docker-file-simple >----------------
+[INFO] Building Eclipse JKube :: Quickstarts :: Maven :: Dockerfile :: Simple 1.0.0-SNAPSHOT
+[INFO] --------------------------------[ jar ]---------------------------------
+[INFO] 
+[INFO] --- kubernetes-maven-plugin:1.0.0-SNAPSHOT:resource (default-cli) @ docker-file-simple ---
+[INFO] k8s: jkube-controller: Adding a default Deployment
+[INFO] k8s: jkube-service: Adding a default service 'docker-file-simple' with ports [8080]
+[INFO] k8s: jkube-healthcheck-spring-boot: Adding readiness probe on port 8080, path='/actuator/health', scheme='HTTP', with initial delay 10 seconds
+[INFO] k8s: jkube-healthcheck-spring-boot: Adding liveness probe on port 8080, path='/actuator/health', scheme='HTTP', with initial delay 180 seconds
+[INFO] k8s: jkube-revision-history: Adding revision history limit to 2
+[INFO] k8s: validating /home/user/00-MN/projects/forks/jkube/quickstarts/maven/docker-file-simple/target/classes/META-INF/jkube/kubernetes/docker-file-simple-deployment.yml resource
+[INFO] k8s: validating /home/user/00-MN/projects/forks/jkube/quickstarts/maven/docker-file-simple/target/classes/META-INF/jkube/kubernetes/docker-file-simple-service.yml resource
+[INFO] 
+[INFO] --- kubernetes-maven-plugin:1.0.0-SNAPSHOT:apply (default-cli) @ docker-file-simple ---
+[INFO] k8s: Using Kubernetes at https://172.17.0.3:8443/ in namespace default with manifest /home/user/00-MN/projects/forks/jkube/quickstarts/maven/docker-file-simple/target/classes/META-INF/jkube/kubernetes.yml 
+[INFO] k8s: Using namespace: default
+[INFO] k8s: Updating a Service from kubernetes.yml
+[INFO] k8s: Updated Service: target/jkube/applyJson/default/service-docker-file-simple-1.json
+[INFO] k8s: Updating Deployment from kubernetes.yml
+[INFO] k8s: Updated Deployment: target/jkube/applyJson/default/deployment-docker-file-simple-2.json
+[INFO] k8s: HINT: Use the command `kubectl get pods -w` to watch your pods start up
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  7.688 s
+[INFO] Finished at: 2020-06-11T08:01:21+02:00
+[INFO] ------------------------------------------------------------------------
+$ kubectl get pods
+NAME                                      READY   STATUS             RESTARTS   AGE
+docker-file-simple-686cddc6b9-lvgtv       0/1     Running            0          84s
+$ kubectl logs pod/zero-config-dockerfile-686cddc6b9-lvgtv
```

Comment:
Should we add additional steps for adding `Deployment` and `Service` fragment for port configuration here?

Reply:
I'm not following
Service config is added in pom.xml configuration (36 & 37), this enabled the generation of service and deployment YAML manifests. Do you mean to describe that in the readme too?

Reply:
Umm, yeah. Does generated `Deployment` manifest contains `ContainerPort`?

Reply:
> Does generated Deployment manifest contains ContainerPort?

Nope, Deployment manifest doesn't contain the containerPort entry(ies). This field is purely [informational](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#container-v1-core), so as the service is already there and quickstart will work either way as expected, maybe adding this extra configurations can be misleading (we already opened https://github.com/eclipse/jkube/issues/240 to take care of this > deployment will be 100% compliant once fixed).

> Umm, yeah.

I'll add a paragraph with the explanation.